packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -164,11 +164,11 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
         className={ mergeStyles(
           'ms-Nav-link' + link.onClick && 'ms-Nav-linkButton',
           styles.link,
-          link.onClick && styles.buttonEntry,
+          link.onClick && !link.forceAnchor && styles.buttonEntry,
           this._hasExpandButton && 'isOnExpanded') as string
         }
         styles={ buttonStyles }
-        href={ link.url }
+        href={ link.url || (link.forceAnchor ? 'javascript:' : undefined) }
```

Comment:
why just javascript: and not javascript:void(0) ?

Reply:
I just restored what was there before my refactoring. If `renderLink` was called without any url, this is the string that was added to `href`

![image](https://user-images.githubusercontent.com/3417264/32066922-abf2e556-ba81-11e7-853e-1eeaaa0afb99.png)

I can change it to `javascript:void(0)` if it makes more sense!

Reply:
I'm good reverting to the original code. Usually safer in case there was a reason for doing it that way :)